asv_bench/asv.conf.json

```diff
@@ -50,6 +50,7 @@
         "numexpr": [],
         "pytables": [null, ""],  // platform dependent, see excludes below
         "pyarrow": [],
+        "pytorch": [],
```

Comment:
the benchmark still runs with this

https://www.kaggle.com/code/marcogorelli/asv-pandas-example/notebook?scriptVersionId=111530834

it doesn't currently run in CI because the job that runs asv uses ``--python same``

https://github.com/pandas-dev/pandas/blob/2a2daf787c5bf4015729f6660d189389d3d7dc47/.github/workflows/code-checks.yml#L128

I'll see what can be done about this

Perhaps `pytorch` can just be appended to the environment before running the asv benchmarks